free-2.md

```diff
@@ -58,11 +75,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I beleive that a person working on this project would spend the most time with system aspects but It would depend on how much natural language is used for the laguage itself. If the laguage is 100% natural laguage driven there will have to be many componets used for laguange analysis. The sysytem processes themselves will be roughly straight forward. build a data structire to score stats per game per player, use those stats to make calculations on averages, team production, and other analysis. Then present all findings per game to the user once an end game command is enetered. Additionally allow for data to be uploaded or saved for people like familly memebers or program websites to see. This process would grow more difficult as you add more sports but could be failry simple for one or two sports. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think this idea could be as big as one wants to make it. It could quite simply be for the average sports dad at their kids little leage game allowing for the parent to add quick stats about the kids playing to the system. However it could be even bigger in terms of collegiate and maybe even profesional use. 
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+I think this is a good idea for a project because it has aplications to the typical human being within the sports domain. Additionally i think this idea could be a decent project to accomplish varring aspects depending on the scale that the class deams as good enough to complete within a time frame. This idea may not be the best idea because it is mostly applicable to the atheltic world and does not have much growth beyond that. 
```

Comment:
I think that the limited growth aspect might actually make it a good project - the domain is clear and tight.